src/UpdatedPackagesCollector.js

```diff
@@ -151,6 +151,7 @@ export default class UpdatedPackagesCollector {
       return file.replace(folder + path.sep, "");
     });
 
+    // TODO: Ugh... something about this.
```

Comment:
So... config for `ignore` and `scope` filtration is currently either on the CLI or via `publishConfig` or `bootstrapConfig`.  I'd like to generalize that to `command.<command>.{ignore,scope}` or something like that.  But... this is filtering in a helper class that is used by multiple commands.  Need to figure out what to do about that.  Really, it's nice to have `asini updated` and `asini publish` _agree_... so maybe the `command.<command>.{ignore,scope}` style isn't the right answer?